ci_framework/roles/manage_secrets/tasks/dataplane_ssh.yml

```diff
@@ -0,0 +1,82 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Check if secret exists
+  environment:
+    PATH: "{{ cifmw_path }}"
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+  failed_when: false
+  register: _secret_check
+  ansible.builtin.command:
+    cmd: >-
+      oc get secret {{ cifmw_manage_secrets_dataplane_ssh_metaname }}
+      -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }}
+
+- name: Assert secret does not exist
+  ansible.builtin.assert:
+    that:
+      - _secret_check.rc != 0
+    msg: >-
+      Looks like {{ cifmw_manage_secrets_dataplane_ssh_name }} exists.
+      Please delete it first calling
+      "oc delete secret {{ cifmw_manage_secrets_dataplane_ssh_name }}"
```

Comment:
Shouldn't this be deleting `{{ cifmw_manage_secrets_dataplane_ssh_metaname }}` instead of `{{ cifmw_manage_secrets_dataplane_ssh_name }}`?

Reply:
woops, yes, good catch.